language/Turkish/strings.po

```diff
@@ -2441,7 +2443,7 @@ msgstr ""
 #, c-format
 msgctxt "gfxview"
 msgid " %1$d×%2$d origin (%3$d %4$d)"
-msgstr ""
+msgstr " %1$d×%2$d köken (%3$d %4$d)"
```

Comment:
This is “origin” in the sense of the origin of a coordinate system.  From a quick look at Wikipedia, it suggests it may be called “sıfır noktası” or similar in Turkish?

Reply:
you're right, fixed it as "sıfır noktası"